### ISSUE TITLE
Rat commands do not work (Follow me, gather items, etc.)

### ISSUE THREAD

**Issue by Hermsmegerms** [Alex-the-666/Rats#577]
Running all the mods 6 server 1.5.1
Minecraft version 1.16.5
Rats version is 7.5.1
Ratlantis version  1.0.0

When attempting to give a tamed rat any command except for stay here, they simply rush over to my cow pen and stare at the cow. follow me does nothing, hunt animals does nothing, gather items does nothing. If i pick them up and walk around then drop them the second they hit the ground they rush over to my cow pasture and just stare at the cow, regardless of command settings. 

I am the host of this server so I can modify configs or collect logs and such if need be.
I am really hoping its just something I am doing wrong here or a config not set up correctly

**Comment by Hermsmegerms** [comment]
After more testing its only freshly tamed rats that seem to have this issue, if i log out of the server then log back in the previously tamed rats will respond correctly to commands but the new tames will not.

**Comment by Hermsmegerms** [comment]
This seems to go deeper. When giving them mining upgrades or quarry upgrades they will attempt the job for about 10 seconds then just go idle again. and stand there doing nothing. I made sure they had a chest or a quarry block to drop items into but they seem to not want to work. dang lazy rats lol

**Comment by KevinDaGame** [comment]
I have the same issue, have you found any solution?

**Comment by Vintschente** [comment]
*push*

Can confirm this behaviour in version 7.1.1. After taming them you need to relog.
Also the quarry works sometimes for some blocks and sometimes for some layers of blocks. THey are doing the mining movement with their hands but no block is getting mined.

**Comment by jelliott7** [comment]
Push

Issue is present for me as well. Only present on MP server (self hosted). SP worlds rats work as intended with commands. MP they do not do anything except "follow me" and "stay here". On follow me, they wont attack mobs that aggro me either.

**Comment by TellyO3** [comment]
I can confirm that none of the rat commands seem to work correctly other than follow, wander and stay here in multiplayer.

**Comment by KittehMei** [comment]
My boyfriend and I have just installed this mod and have noticed this issue as well. Newly tamed rats don't move at all and don't do any jobs. They just sit in one spot. I did attempt to get the rat to do the gathering command and he'd shuffle towards the dropped item and then stop. Sometimes it picked it up, other times it didn't. 

Me personally relogging didn't change anything, but once we restarted the server itself (my boyfriend is hosting on a server box) the rat started following normally. I also tested the gathering command which seems to work for me as well. Though we'll have to see how it goes since we've only been playing with it for a short while.

(We are currently also using FTB Endeavour 1.1.1 if that is any help, we added this rat mod in this evening).
